src/snowcli/cli/snowpark/package/commands.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+import logging
+from pathlib import Path
+
+import typer
+
+from snowcli.cli.common.decorators import global_options
+from snowcli.cli.common.flags import DEFAULT_CONTEXT_SETTINGS
+from snowcli.cli.snowpark.package.manager import PackageManager
+
+app = typer.Typer(
+    name="package",
+    context_settings=DEFAULT_CONTEXT_SETTINGS,
+    help="Manage custom Python packages for Snowpark",
+)
+log = logging.getLogger(__name__)
+
+
+@app.command("lookup")
+@global_options
+def package_lookup(
+    name: str = typer.Argument(..., help="Name of the package"),
+    install_packages: bool = typer.Option(
+        False,
+        "--yes",
+        "-y",
+        help="Install packages that are not available on the Snowflake anaconda channel",
+    ),
+    **kwargs,
+):
+    """
+    Checks if a package is available on the Snowflake anaconda channel.
+    In install_packages flag is set to True, command will check all the dependencies of the packages
+    outside snowflake channel.
+    """
+    log.info(PackageManager().lookup(name, install_packages))
```

Comment:
I think we should return this information instead of log. This message is the result of this operation

Reply:
Sure. That`s exactly what we talked about - what exactly we want to return. Original function sometimes logges, sometimes returned a string.
So, we should return a string and use @with_output?

Reply:
Yup. And I would consider using discussion from https://github.com/Snowflake-Labs/snowcli/pull/304#discussion_r1288076864 to implement the if here so `lookup` return result but the command decided what meesage should be returned

Reply:
Done